quodlibet/po/fr.po

```diff
@@ -1034,217 +1025,216 @@ msgstr "[option]"
 msgstr "Rechercher dans votre bibliothèque de musique"
 
-#: ../quodlibet/cli.py:124 ../quodlibet/cli.py:136 ../quodlibet/cli.py:140
-#: ../quodlibet/cli.py:142
+#: ../quodlibet/cli.py:125 ../quodlibet/cli.py:137 ../quodlibet/cli.py:141
+#: ../quodlibet/cli.py:143
 msgid "query"
-msgstr "recherche"
+msgstr "requête"
 
-#: ../quodlibet/cli.py:125
+#: ../quodlibet/cli.py:126
 msgid "Play a file"
 msgstr "Lire un fichier"
 
-#: ../quodlibet/cli.py:125 ../quodlibet/cli.py:136 ../quodlibet/cli.py:142
-#, fuzzy
+#: ../quodlibet/cli.py:126 ../quodlibet/cli.py:137 ../quodlibet/cli.py:143
 msgctxt "command"
 msgid "filename"
 msgstr "nom de fichier"
 
-#: ../quodlibet/cli.py:126
+#: ../quodlibet/cli.py:127
 msgid "Rate the playing song"
-msgstr "Noter la chanson en cours de lecture"
+msgstr "Évaluer le titre en cours de lecture"
 
-#: ../quodlibet/cli.py:127
+#: ../quodlibet/cli.py:128
 msgid "Set the current browser"
 msgstr "Définir le navigateur courant"
 
-#: ../quodlibet/cli.py:128
-#, fuzzy
+#: ../quodlibet/cli.py:129
 msgid "Stop after the playing song"
-msgstr "Noter la chanson en cours de lecture"
+msgstr "Arrêter après ce titre"
 
-#: ../quodlibet/cli.py:129
+#: ../quodlibet/cli.py:130
 msgid "Open a new browser"
 msgstr "Démarrer un nouveau navigateur"
 
-#: ../quodlibet/cli.py:130
+#: ../quodlibet/cli.py:131
 msgid "Show or hide the queue"
-msgstr "Montrer ou cacher la file"
+msgstr "Afficher ou cacher la file de lecture"
 
-#: ../quodlibet/cli.py:132
-#, fuzzy
+#: ../quodlibet/cli.py:133
 msgid "Show or hide the main song list (deprecated)"
-msgstr "Montrer ou cacher la liste principale des chansons"
+msgstr "Afficher ou cacher la liste principale des titres (abandonné)"
 
-#: ../quodlibet/cli.py:133
+#: ../quodlibet/cli.py:134
 msgid "Filter on a random value"
 msgstr "Filtrer au hasard"
 
-#: ../quodlibet/cli.py:133
-#, fuzzy
+#: ../quodlibet/cli.py:134
 msgctxt "command"
 msgid "tag"
-msgstr "Étiquette"
+msgstr "étiquette"
 
-#: ../quodlibet/cli.py:134
+#: ../quodlibet/cli.py:135
 msgid "Filter on a tag value"
 msgstr "Filtrer sur la valeur d'une étiquette"
 
-#: ../quodlibet/cli.py:134
+#: ../quodlibet/cli.py:135
 msgid "tag=value"
 msgstr "étiquette=valeur"
 
-#: ../quodlibet/cli.py:135
+#: ../quodlibet/cli.py:136
 msgid "Enqueue a file or query"
-msgstr "Mettre un fichier ou le résultat d'une recherche dans la file"
+msgstr ""
+"Mettre un fichier ou le résultat d'une recherche dans la file de lecture"
 
-#: ../quodlibet/cli.py:137
+#: ../quodlibet/cli.py:138
 msgid "Enqueue comma-separated files"
 msgstr ""
+"Mettre des fichiers séparés par un point-virgule dans la file de lecture"
 
-#: ../quodlibet/cli.py:138 ../quodlibet/util/tags.py:146
+#: ../quodlibet/cli.py:139 ../quodlibet/util/tags.py:147
 msgid "filename"
 msgstr "nom de fichier"
 
-#: ../quodlibet/cli.py:139
+#: ../quodlibet/cli.py:140
 msgid "Print filenames of results of query to stdout"
-msgstr ""
+msgstr "Afficher le résultat des requêtes en utilisant stdout"
```

Comment:
"Afficher les résultats de la requête sur la sortie standard stdout"
And "requête" or "recherche" ? I don't know...